applications/Plutonication.md

```diff
@@ -0,0 +1,430 @@
+
+- Our project is comparable to WalletConnect, which was also our inspiration.
+When we started making our first prototypes in February, WalletConnect was not available in the Polkadot Ecosystem yet.
+Even thought they have taken a lot of time and had a lot more experience then us, they were unable to make quick and elegant deliveries.
+WalletConnect still is not supported by most of the dApps. We think we can do better. Actually, we already did.
+
+- We made a Plutonication Extension, which already allows you to interact with existing dApps, even though they have not implemented the Plutonication standard directly. This can be a perfect middle ground during the transition of popularizing the Plutonication. Even if the user wanted to use a new niche dApp, they can do so with Plutonication.
+
+- Wallet connect is also only supported in javascript. We want to make Plutonication available in more languages, including a very popular C# language, which is mostly used for game development. We will make web3 gaming possible on game consoles, thanks to the Plutonication.
+
+#### Who is your target audience (parachain/dapp/wallet/UI developers, designers, your own user base, some dapp's userbase, yourself)?
+
+- dApps developers - To integrate the Plutonication in their dApps. We will ensure that the developers will receive a good documentation.
+
+- Wallet developers - We are welcoming other wallets to use the Plutonication. We would like to help them make this possible.
+
+- Users - In the end, it will be mainly used by mobile focused users. They will be able to interact with web3 apps on unusual platforms, like game consoles, smartwatches ...
+
+#### What need(s) does your project meet?
+
+- You are unable securely interact with dApps on gaming consoles, smartwatches and other unusual platforms. Without Plutonication, you would have to expose your private key, which is very unsafe.
+
+- You can also interact with existing web dApps with your mobile wallet. Again, you do not need to expose your private key to multiple places (in this case browser extension wallet), which would be very unsafe.
+
+#### Are there any other projects similar to yours in the Substrate / Polkadot / Kusama ecosystem?
+
+- Yes, WalletConnect.
+
+#### If so, how is your project different?
+
+- We support more programming languages. We also have a browser extension that enables Plutonication on existing dApps.
+
+- WalletConnect also mainly focuses on Ethereum ecosystem. We are focusing only on Polkadot.    
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+#### Name of team leader: 
+
+Rostislav Litovkin
+
+#### Names of team members:
+
+Valentina Gómez
+
+Dušan Jánsky
+
+### Contact
+
+- **Contact Name:** Rostislav Litovkin
+- **Contact Email:** rostislavlitovkin@gmail.com
+- **Website:** https://github.com/cisar2218/Plutonication
+
+### Legal Structure
+
+- **Registered Address:** None
+- **Registered Legal Entity:** None
```

Comment:
Please provide at least an address. If you prefer not to do so publicly, feel free to [email us](mailto:grants@web3.foundation).